index.js

```diff
@@ -7,14 +23,258 @@ function ask(questionText) {
   });
 }
 
-start();
+//define player object literal
+
+let Player = {
+  name: "",
+  mood: "",
+  inventory: ["first aid kit", "nalgene", "peanut butter M&Ms (melted)"]
```

Comment:
I noticed these items are not created as instances of the Item class constructor. I am wondering if that is a piece in progress or if it was intentional?